ivadomed/utils.py

```diff
@@ -33,6 +34,38 @@ def get_results(self):
         return res_dict
 
 
+def save_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis):
+    nib_ref = nib.load(fname_ref)
```

Comment:
it would be useful to add a docstrings as this function seems to do more than just saving a nifti file (otherwise calling nibabel would suffice)